components/components.css

```diff
@@ -55,4 +55,10 @@
      --badge-count-color : #eeebeb;
      --badge-count-border-radius: 50%;
      /* Badges End */
+
+     /* Inputs */
+    --input-font-size:1rem;
+    --text-grey:#808080;
+    --input-width:300px;
```

Comment:
Try to avoid using fixed width for input, you may need it in different places where you require different width.

Reply:
Yes , got it . I'll make the changes
Thanks for correcting me